src/triggers/CBL_Attitud_UpdateDataCbl2.trigger

```diff
@@ -0,0 +1,51 @@
+trigger CBL_Attitud_UpdateDataCbl2 on CBL_Attitude__c (after insert) {
+	List<CBL_Attitude__c> cbls_att = new List<CBL_Attitude__c>();
+    Set<Id> cbls_Ids = new Set<Id>();
+    Set<Id> LAC_FD_Ids = new Set<Id>();
+    
+    //get a list with Baseline id related to attitude rows inserted
+    for (CBL_Attitude__c newRow : Trigger.new) {
+        cbls_Ids.add(newRow.Baseline__c);
+    }
+    //get a list with attitude rows completed (Baseline__r.Farmer__c)
+    cbls_att = [SELECT Id, Baseline__c, Baseline__r.Farmer__c, chooseToBeFarmer__c, destinedToBeFarmer__c, discussMethodsWOthers__c, effortInAgriculturePays__c, farmersLifeWillBeBetter__c, futureCoffeePlans__c, happyToBeFarmer__c, limitationsCauses__c, neighborsProductivityLvl__c, payForAgriAdvise__c, productivityImprovmntLimitations__c, productivityTrend5Years__c, takeAdvise__c, wishChildrenBecomeFarmers__c, worthInvestingMoneyTimeAgr__c, Submission__c 
```

Comment:
I think this can be made batch safe. The idea is to try and process in batches of 200:

```
for (List<MyObj> objs : [SELECT .... from MyObj where id :IN ]) {
    for (MyObj obj: objs) {
        // process obj here...
    }
}
```